src/release/breaking-changes/index.md

```diff
@@ -51,6 +50,7 @@ release, and listed in alphabetical order:
 * [Removed `ignoringSemantics`][] properties
 * [Deprecated `RouteInformation.location`][] and its related APIs
 * [Updated EditableText scroll into view behavior][]
+* [Customize tabs alignment using the new `TabBar.tabAlignment` property][]
```

Comment:
Looks like the reference resolution should be moved on line 39 as well?

Reply:
Oops, late night edit. Fixed.